e2e/support-us.spec.js

```diff
@@ -0,0 +1,39 @@
+const { takeScreenshot } = require("./helpers");
+
+describe("e2e/support-us", () => {
+  beforeEach(async () => {
+    await device.reloadReactNative();
+  });
+
+  afterEach(async () => {
+    await takeScreenshot();
+  });
+
+  it("should display links to relevant sections", async () => {
+    await element(by.id("support-us-tab-button")).tap();
+
+    await expect(element(by.id("support-us-donate-button"))).toBeVisible();
+    await expect(element(by.id("support-us-volunteer-button"))).toBeVisible();
+    await expect(element(by.id("support-us-merchandise-button"))).toBeVisible();
+
+    await element(by.id("support-us-main-content")).scrollTo("bottom");
```

Comment:
This is the scrolling I can't get working, as referenced in the TODO section of the PR description...

Reply:
I wonder of the scroll does not work because the "tap" happens on the tiny shadow we added on top the tab navigation. This shadow is positioned absolute above the scroll view. My fingers are not small enough to verify this on a physical device but I can reproduce this with the mouse cursor in an emulator. Trying to scroll by tapping on this shadow does not work. 🤔

Reply:
Yeah, this is the reason. We fixed it by adding `pointerEvents="none"` to the shadow view.